### PR TITLE
DEV-4655: quizz-app Readme

### PR DESCRIPTION
**Pull request by MSilva95**
fixing structure of the readme
adding more instructions to make the project easier to achieve

### CODE REVIEW

File: subjects/mobile-dev/quizz-app/README.md [hunk 1]
```diff
@@ -1,32 +1,23 @@
...
-Develop a [**stateful Quizz app**](https://docs.flutter.dev/development/ui/interactive) that displays categories that you can choose questions from.
-Possible categories: history, pop culture, math, geography, etc (appropriate name, image must be added to all categories).
+You need to develop a fully functional [**stateful Quizz app**](https://docs.flutter.dev/development/ui/interactive) in Flutter that allows users to select a **category** and answer **true or false questions** related to the selected category.
 
-After tapping on the category the new route must show a corresponding image to the category, the question itself, true/false buttons.
+**Features:**
 
-After the question has been answered, you must let the user know if the question was answered correctly.
+- The app needs to feature at least 5 different categories that the users can select from the main page, i.e, history, culture, math, geography, etc.
+- The app needs to feature at least at least 10 questions to each `quizz`.
```
Comment (nprimo) [inline subjects/mobile-dev/quizz-app/README.md:10]: In the audit it is required to have 10 questions for each category: here it is required to have it for each `quizz` - is this equivalent/intended to be like this?

File: subjects/mobile-dev/quizz-app/README.md [hunk 2]
```diff
@@ -1,32 +1,23 @@
...
+- Appropriate name and images must be added to all of the selected categories. (follow the examples bellow)
```
Comment (nprimo) [inline subjects/mobile-dev/quizz-app/README.md:11]: My grammar checker suggest to change `all of the` to `all the`

**Review by nprimo — COMMENTED** [review]
There are some minor changes that I have suggested in line. Feel free to implement them or not. 
Overall it looks good to me: there is only one point (marked as **MORE RELEVANT**) that might be confusing (in relation to the audit)

File: subjects/mobile-dev/quizz-app/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/mobile-dev/quizz-app/README.md]: i've modified the sentence to be more clear

[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/mobile-dev/quizz-app/README.md]: done

File: subjects/mobile-dev/quizz-app/README.md (hunk 2 above)
Comment (MSilva95) [reply subjects/mobile-dev/quizz-app/README.md]: done

**Pull request merged**